PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py

```diff
@@ -63,6 +64,7 @@ def miniAOD_customizeMETFiltersFastSim(process):
 phase2_common.toReplaceWith( Flag_trkPOG_manystripclus53X, cms.Path() )
 phase2_common.toReplaceWith( Flag_trkPOG_toomanystripclus53X, cms.Path() )
 phase2_common.toReplaceWith( Flag_trkPOGFilters, cms.Path(~logErrorTooManyClusters) )
+phase2_common.toReplaceWith( Flag_ecalBadCalibFilter, cms.Path(ecalBadCalibFilter) )
```

Comment:
sorry, it should be run2_common

Reply:
Should I then also import run2_common in addition to:

from Configuration.Eras.Modifier_phase2_common_cff import phase2_common

?

Thank you

Reply:
yes, please.